src/battle_animation.cpp

```diff
@@ -311,6 +315,92 @@ void BattleAnimationBattle::ShakeTargets(int str, int spd, int time) {
 	}
 }
 
+BattleAnimationBattler::BattleAnimationBattler(const lcf::rpg::Animation& anim, std::vector<Game_Battler*> battlers, bool only_sound, int cutoff_frame, bool set_invert) :
+	BattleAnimation(anim, only_sound, cutoff_frame), battlers(std::move(battlers))
+{
+	invert = set_invert;
+}
```

Comment:
If I understand this correctly a BattleAnimationBattler is this Animation2 stuff from RPG2k3 (as alternative to CBA), right?

Reply:
You are correct. The BattleAnimationBattler class is meant to handle battle animation sprite battlers.